metrics/storage/webtooling.sh

```diff
@@ -74,55 +131,78 @@ function main() {
 		exit 1
 	fi
 
-	containers=()
-	# Check tools/commands dependencies
-	local cmds=()
-	cmds+=("docker")
+	local i=0
+	local containers=()
+	local cmds=("docker")
+	local not_started_count=$NUM_CONTAINERS
 
 	restart_containerd_service
+	# Check tools/commands dependencies
+	clean
 	init_env
 	check_cmds "${cmds[@]}"
 	check_ctr_images "$IMAGE" "$DOCKERFILE"
-
 	metrics_json_init
-
 	save_config
 
+	echo "INFO: creating $NUM_CONTAINERS containers"
```

Comment:
Please use `info "Creating..."` instead of `echo INFO:...`

Reply:
@marcel-apf  thank you, changes applied